src/p_settings.hxx

```diff
@@ -822,16 +822,19 @@ private:
 
 class AspectRatioSetting final : public Settings::IntSetting {
 public:
-    enum Kind { FULL, WIDE, ULTRAWIDE };
+    enum Kind { FULLOPENMATTE, FULL, WIDE, ULTRAWIDE };
 
     AspectRatioSetting(const char *name) : IntSetting(name, &AspectRatioSetting::sAspectRatioValue) {
-        mValueRange     = {0, 2, 1};
+        mValueRange     = {0, 2, 1, 1};
```

Comment:
mValueRange is a struct with the following properties: `{start, stop, step}`, with this range being inclusive. Rather than `{0, 2, 1, 1}` which is incorrect and won't compile, do `{0, 3, 1}`.